extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -223,6 +280,21 @@ export class ProjectsController {
 
 	//#region Helper methods
 
+	public getDeployDialog(project: Project): DeployDatabaseDialog {
+		return new DeployDatabaseDialog(this.apiWrapper, project);
+	}
+
+	private static async getService(extensionId: string): Promise<mssql.IDacFxService> {
+		const ext = extensions.getExtension(extensionId);
```

Comment:
Why are you passing in the extension ID? Do you ever expect a different extension to supply the DacFX service?  Considering you cast to mssql.IExtension that seems unlikely